source/MapPlanetCard.cpp

```diff
@@ -43,6 +47,11 @@ MapPlanetCard::MapPlanetCard(const StellarObject &object, unsigned number, bool
 	hasSpaceport = planet->HasSpaceport();
 	hasShipyard = planet->HasShipyard();
 	hasOutfitter = planet->HasOutfitter();
+	governmentName = planet->GetGovernment()->GetName();
+	if(systemGovernmentName.empty())
+		systemGovernmentName = planet->GetSystem()->GetGovernment()->GetName();
+	if(planet->GetGovernment()->GetName() != "Uninhabited" && governmentName != systemGovernmentName)
+		hasGovernments = true;
```

Comment:
```suggestion
	governmentName = planet->GetGovernment()->GetName();
	if(systemGovernmentName.empty())
		systemGovernmentName = planet->GetSystem()->GetGovernment()->GetName();
	if(governmentName != "Uninhabited" && governmentName != systemGovernmentName)
		hasGovernments = true;
```
Isn't this what you just assigned to this variable?